cmd/evm/staterunner.go

```diff
@@ -31,11 +34,34 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+var (
+	ForkFlag = &cli.StringFlag{
+		Name:     "subtest.fork",
```

Comment:
as we're in `geth evm statetest`, I think `--statetest.fork` and `--statetest.index` etc would make more sense.